validation/mock_metric_validation.go

```diff
@@ -1,15 +1,27 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 package validation
 
 import (
-	"fmt"
 	"reflect"
 
 	"google.golang.org/genproto/googleapis/monitoring/v3"
-	"google.golang.org/genproto/googleapis/rpc/errdetails"
 )
 
 const (
-	missingFieldMsg = "%v must contain requried %v field"
+	missingFieldMsg = "%v must contain required %v field"
```

Comment:
We can move this field into the shared file as well.

Reply:
good point, done